config/server_test.go

```diff
@@ -0,0 +1,48 @@
+package config_test
+
+import (
+	"testing"
+
+	"github.com/NYTimes/gizmo/config"
+)
+
+func TestServerVanillaConfig(t *testing.T) {
+	server := config.Server{}
+	portBound := server.PortBound()
+	logsPopulated := server.LogsPopulated()
+	healthCheckSet := server.HealthCheckSet()
+
+	if portBound != false {
+		t.Errorf("Expected portBound to be false, got %v", portBound)
+	}
+
+	if logsPopulated != false {
+		t.Errorf("Expected logsPopulated to be false, got %v", logsPopulated)
+	}
+
+	if healthCheckSet != false {
+		t.Errorf("Expected healhCheckSet to be false, got %v", healthCheckSet)
+	}
+}
+
+func TestServerVanillaConfig(t *testing.T) {
+	server := new(config.Server)
+	server.HTTPPort = 80
+	server.HTTPAccessLog = "LOG"
+	server.HealthCheckType = "PINGDOM"
+	portBound := server.PortBound()
```

Comment:
It looks like you're missing the implementations of a few methods (PortBound, LogsPopulated, HealthCheckSet)

Not sure about adding these methods just to check for empty/nil values. It seems like it may add some unneeded misdirection, but I'll wait to see the full implementation before rejecting them.